app/assets/stylesheets/table-category-column.scss

```diff
@@ -0,0 +1,37 @@
+p.table-category,
+p.table-category-vacant {
+  padding: 4px 1px 1px;
+  color: #005ea5;
+  border: 2px solid;
+  margin: 0 1px 2px;
+  background: white;
+  white-space: nowrap;
+  text-transform:none;
+  text-align:center;
+}
+
+p.table-category-vacant {
+  visibility: hidden;
+  &.table-category-preview {
+    visibility: visible;
+  }
+}
+td.govuk-table__cell.tag-cell {
+  text-align:center;
+}
+
+@media (min-width: 555px) {
+  p.table-category, p.table-category-vacant {
+    padding: 4px 3px 1px;
+  }
+}
+@media (min-width: 655px) {
+  p.table-category, p.table-category-vacant {
+    padding: 4px 8px 1px;
+  }
+}
+@media (min-width: 705px) {
+  p.table-category, p.table-category-vacant {
+      white-space: nowrap;
+  }
+}
```

Comment:
The styling is specified to the `p` tag to push it heigher up the heirarchy than the GDS out of the box style, which otherwise gets applied.  I can (and have) removed it from where it isn't needed.  

Other changes made as per your comments.